CONTRIBUTING.md

```diff
@@ -4,22 +4,74 @@ First off, thanks for taking the time to contribute!
 
 The following is a set of guidelines and helpful information for contributing to the 3D Artist's Handbook, which is hosted on [GitHub](https://github.com/sidney-eliot/3d-artists-handbook). These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
 
-#### Table Of Contents
-
-ToDo
+## Table of Contents
+
+- [Code of Conduct](#code-of-conduct)
+- [What Should I Know Before I Get Started?](#what-should-i-know-before-i-get-started)
+   - [Prerequisites](#prerequisites)
+   - [Choosing What to Work On](#choosing-what-to-work-on)
+- [How Can I Contribute?](#how-can-i-contribute)
+   - [Test Locally Before You Push](#test-locally-before-you-push)
+- [Styleguides](#styleguides)
+   - [Handbook Pages Styleguide](#handbook-pages-styleguide)
+      - [General](#general)
+      - [Specific](#specific)
+- [Additional Note](#additional-notes)
 
 ## Code of Conduct
+
 This project and everyone participating in it is governed by the [3D Artist's Handbook Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code.
 
-## What should I know before I get started?
+## What Should I Know Before I Get Started?
 
-ToDo
+### Prerequisites
+
+1. You will need to have [Node.js](https://nodejs.org/en/download/package-manager), and thus `npm` available on your
+   terminal of choice.
+
+   > [!NOTE]
+   > Installing the latest version should work without much problem. However, just in case as a reminder, as of 
+   > 2024-12-10, Quartz 4 needs _at least_ Node v20 and up, as well as `npm` v9.3.1 and up to work.
+   >
+   > By the time you read this, the information contained here may be outdated, for up to date information on Quartz's
+   > dependencies, please consult Quartz's own documentation, which should be found [here](https://quartz.jzhao.xyz/).
```

Comment:
I believe Obsidian uses the same blockquote syntax as GitHub does it in its own markdown, [GFM (GitHub Flavored Markdown)](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts).

Like this ⬇️:
```
> [!NOTE]
> Like this.
```
> [!NOTE]
> Like this.

Although, if we really wanted the `CONTRIBUTING.md` and the like to use only the most basic vanilla markdown possible, that is a choice too. That said, we would have to get rid of tables since it is technically an extension, as stated in the [GFM spec](https://github.github.com/gfm/#tables-extension-). Vanilla markdown itself is very limited, as you can see [here](https://www.markdownguide.org/cheat-sheet/), only looking at the *Basic Syntax* section, the rest are extensions that are implemented differently based on the *flavor* of the markdown, GFM being only one of them, based on [CommonMark](https://commonmark.org/), another widely-known flavor. You can also see the original vanilla markdown spec by the creator John Gruber, [here](https://daringfireball.net/projects/markdown/syntax).